x-pack/plugins/observability/public/components/app/observability_status/observability_status.stories.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { ObservabilityStatusBoxes, ObservabilityStatusProps } from './observability_status_boxes';
+
+export default {
+  title: 'app/ObservabilityStatusBoxes',
+  component: ObservabilityStatusBoxes,
+};
+
+const testBoxes = [
+  {
+    id: 'logs',
+    dataSource: 'Logs',
+    hasData: true,
+    description: 'This is the description for logs',
+    modules: [
+      { name: 'system', hasData: true },
+      { name: 'kubernetes', hasData: false },
+      { name: 'docker', hasData: true },
+    ],
+    integrationLink: 'http://example.com',
+    learnMoreLink: 'http://example.com',
+  },
+  {
+    id: 'metrics',
+    dataSource: 'Metrics',
```

Comment:
I fear that a few of the labels here (like "Metrics" and "Uptime") don't necessarily make sense, especially to new users. I think what we mean with "Metrics" is more or less "Infrastructure monitoring". This should not be a blocker, just food for thought. 
cc @formgeist @danielkhan